compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -2034,6 +2034,170 @@ public void exitStringTemplateLiteral(BallerinaParser.StringTemplateLiteralConte
+            return;
+        }
+
+
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void exitTableQueryExpression(BallerinaParser.TableQueryExpressionContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+    }
+
+    @Override
+    public void enterOrderByClause(BallerinaParser.OrderByClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.startOrderByClauseNode(getCurrentPos(ctx), getWS(ctx));
+
+    }
+
+    @Override
+    public void exitOrderByClause(BallerinaParser.OrderByClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.endOrderByClauseNode(getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void enterGroupByClause(BallerinaParser.GroupByClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.startGroupByClauseNode(getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void exitGroupByClause(BallerinaParser.GroupByClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.endGroupByClauseNode(getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void enterStreamingQueryStatement(BallerinaParser.StreamingQueryStatementContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+    }
+
+    @Override
+    public void exitStreamingQueryStatement(BallerinaParser.StreamingQueryStatementContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+    }
+
+    @Override
+    public void enterHavingClause(BallerinaParser.HavingClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.startHavingClauseNode(getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void exitHavingClause(BallerinaParser.HavingClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.endHavingClauseNode(getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void enterSelectExpression(BallerinaParser.SelectExpressionContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.startSelectExpressionNode(getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void exitSelectExpression(BallerinaParser.SelectExpressionContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        this.pkgBuilder.endSelectExpressionNode(ctx.Identifier().getText(), getCurrentPos(ctx), getWS(ctx));
+    }
+
+    @Override
+    public void enterSelectClause(BallerinaParser.SelectClauseContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+
+        //  this.pkgBuilder.startSelectClauseNode(getCurrentPos(ctx), getWS(ctx));
```

Comment:
Remove commented code and unnecessary spaces

Reply:
fixed